README.md

```diff
@@ -294,6 +294,41 @@ npm run lint
 npm run format
 ```
 
+## 👩‍🎤 Storybook
+
+Storybook helps us build UI components in isolation from our app's business logic, data, and context. That makes it easy to develop hard-to-reach states and save these UI states as stories to revisit during development, testing, or QA.
+
```

Comment:
We totally forgot to document `npm run storybook`, should have another sentence here like

> Starting up the Storybook server with this command will make it accessible under `http://localhost:6006`:
> 
> ```bash
> npm run storybook
> ```

And then I tried that but I get an error when doing so:

<img width="606" alt="Screen Shot 2022-05-06 at 13 23 29" src="https://user-images.githubusercontent.com/90316/167130569-4c5847f4-4b3c-42d9-9962-18ec65193333.png">

Reply:
Just updated the docs. Were you able to make it run? I just retry (cleaning up modules and next cache) and it's working on my end.

You should see this:
<img width="1440" alt="Screen Shot 2022-05-06 at 9 05 44 AM" src="https://user-images.githubusercontent.com/13741335/167137324-a207b2ec-5032-4f8d-8072-42b299b76f68.png">

Reply:
yeah all good, jumping between many PRs locally always gets me in a messed up `node_modules` state when dependency updates are happening in all of them. So all good here

Reply:
Great! 🚀